internal/testing/dochelper/dochelper.go

```diff
@@ -0,0 +1,178 @@
+//
+// Copyright 2022 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dochelper
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"reflect"
+
+	"github.com/guacsec/guac/pkg/handler/processor"
+	"github.com/secure-systems-lab/go-securesystemslib/dsse"
+)
+
+var (
+	// DSSE/SLSA Testdata
+
+	// Taken from: https://slsa.dev/provenance/v0.1#example
+	ite6SLSA = `
```

Comment:
maybe move the examples to a different file within the directory, since it doesn't quite fit into "dochelper"

Reply:
DSSE/SLSA Testdata was moved to this package as the testdata package caused cycle errors when testing with the emitter (as the emitter is used by the assembler).

Reply:
or I could just create the documents at the tests and remove this all together.

Reply:
Moved them all to testdata along with the others. Only for the emitter I will define locally to not cause import cycle error.